inc/commondbtm.class.php

```diff
@@ -2642,7 +2642,9 @@ function showFormHeader($options = []) {
             if ($this->isEntityAssign()) {
                if (isset($params['entities_id'])) {
                   $entity = $this->fields['entities_id'] = $params['entities_id'];
-
+               } else if(isset($this->fields['entities_id'])) {
```

Comment:
```suggestion
               } else if (isset($this->fields['entities_id'])) {
```

Reply:
The last `else` case should be removed I guess.

Reply:
Indeed, removed